### PR TITLE
gapic: in-repo showcase integration

### PR DESCRIPTION
**Pull request by noahdietz**
* add a `showcase` module with (refactored from gapic-generator) Echo Service tests
* update utils/showcase.bash to use in-repo tests

### CODE REVIEW

File: showcase/go.mod [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	github.com/googleapis/gax-go v0.0.0-20181219185031-c8a15bac9b9f // indirect
+	github.com/googleapis/gax-go/v2 v2.0.3 // indirect
+	google.golang.org/api v0.1.0
+	google.golang.org/genproto v0.0.0-20190219182410-082222b4a5c5
+	google.golang.org/grpc v1.18.0
+)
+
+replace cloud.google.com/go/showcase => ./gen/cloud.google.com/go/showcase
+
+replace github.com/googleapis/gapic-showcase => ./gen/github.com/googleapis/gapic-showcase
```
Comment (pongad) [inline showcase/go.mod]: Why do we need this? Shouldn't we just depend on a specific version of `github.com/googleapis/gapic-showcase`?

Comment (pongad) [reply showcase/go.mod]: Oh, it's because we don't actually have that package do we :P

Comment (noahdietz) [reply showcase/go.mod]: Well actually you have a point. This is the base module for `github.com/googleapis/gapic-showcase/server/genproto`. We could depend on the tagged release.

Idk why but i got it in my head that these files should be generated at testing time as well.

Comment (pongad) [reply showcase/go.mod]: Oh right that would make sense.

Comment (noahdietz) [reply showcase/go.mod]: Removed it & the stub generation as well. Depending on the tagged version now.

**Pull request merged**